apps/cms/test/paragraph/content_list_test.exs

```diff
@@ -206,8 +198,8 @@ defmodule CMS.Partial.Paragraph.ContentListTest do
     {"#{k}", [%{"value" => v}]}
   end
 
-  defp cms_field({:terms = k, terms}) do
-    {"field_#{k}", Enum.map(terms, &%{"target_id" => &1})}
```

Comment:
this looks like it used to build a tuple that contained a list of possibly many items, but now it will just contain a list of one item.  Is this the desired change?

Reply:
Correct. The CMS API is being updated to limit each field to only a single value array. To date, nothing the CMS used more than one term -- the JSON response will never have more than one result.